Assets/Scripts/ManagerScripts/CountdownManager.cs

```diff
@@ -17,40 +17,52 @@ private void Awake()
         public TMP_Text countdownText;
         public GameObject countdownUI;
         public AudioSource countdownSound;
-        private bool _shouldCountTime;
+        public bool shouldCountTime;
         private bool _soundPlayed;
+        private bool _gameStart;
 
         private void Start()
         {
-            _shouldCountTime = SceneManager.GetActiveScene().name != "MainMenuScene";
+            shouldCountTime = true;
             countdown = 5f;
+            _gameStart = true;
         }
         
         private void Update()
         {
-            if ( _shouldCountTime && !_soundPlayed)
+            if (shouldCountTime)
             {
-                countingDown = true;
-                countdownSound.Play();
-                _soundPlayed = true;
-            }
-            
-            if (_shouldCountTime && countdown > 0)
-            {
-                countdown -= Time.deltaTime;
-                countdownText.text = (countdown).ToString("0");
-            }
-            else
-            {
-                countdownSound.Stop();
-                countdownUI.SetActive(false);
-                countingDown = false;
+                if (!_soundPlayed)
+                {
+                    countingDown = true;
+                    countdownSound.Play();
+                    _soundPlayed = true;
+                }
+
+                if (countdown > 0)
+                {
+                    SubtractTime(_gameStart);
+                    countdownText.text = (countdown).ToString("0");
+                }
+                else
+                {
+                    countdownSound.Stop();
+                    countdownUI.SetActive(false);
+                    _soundPlayed = false;
+                    _gameStart = false;
+                    countingDown = false;
+                }
             }
         }
 
         public void SetCountdown(float time)
         {
             countdown = time;
+            shouldCountTime = true;
             countdownUI.SetActive(true);
         }
+        
+        private void SubtractTime(bool gameStart) {
```

Comment:
Hol up Im dumb. You can just reference _gameStart directly since it's an attribute of the class. You don't need to pass it in to the method as an argument

Reply:
Fixed